incubator/azuremonitor-containers/templates/omsagent-rbac.yaml

```diff
@@ -21,6 +21,12 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "events", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
+- apiGroups: ["extensions"]
```

Comment:
You might want to future proof this given this API is removed in 1.16 - https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Reply:
yes, we have whole bunch of changes coming here (to support 1.16) - in the upcoming release.We will be updating all these are part of that.